#!/usr/bin/env python
PACKAGE = "ur3_controls"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add("Dynamic_Reconfigure",   bool_t,   0, "Dynamic Reconfigure",  False)

gen.add("Paint_enable",   bool_t,   0, "Base Enable",  False)
gen.add("R_lower",    int_t,    0, "R", 100,  0, 255)
gen.add("G_lower",    int_t,    0, "G", 100,  0, 255)
gen.add("B_lower",    int_t,    0, "g", 20,  0, 255)

gen.add("R_upper",    int_t,    0, "R", 255,  0, 255)
gen.add("G_upper",    int_t,    0, "G", 255,  0, 255)
gen.add("B_upper",    int_t,    0, "g", 30,  0, 255)

gen.add("Color_paint",    int_t,    0, "Base Position", 0,  0, 5)
gen.add("Paint_clear",   bool_t,   0, "clear",  False)

exit(gen.generate(PACKAGE,  "ur3_controls", "DH"))
